addons/dark-www/scrollbar.css

```diff
@@ -1,20 +1,20 @@
 body:not(.sa-body-editor)::-webkit-scrollbar,
 body:not(.sa-body-editor) ::-webkit-scrollbar {
-  width: 16px;
-  height: 16px;
+  width: 17px;
+  height: 17px;
```

Comment:
@mxmou Just wondering why? 😂

Reply:
17px is the width of Chromium's default scrollbar for some reason, so using the same width prevents the entire page from shifting slightly when enabling or disabling the setting.